.eslintrc.cjs

```diff
@@ -0,0 +1,12 @@
+/** @type {import('@typescript-eslint/utils').TSESLint.Linter.Config} */
+const config = {
+  extends: ['upleveled'],
+  overrides: [
+    {
+      files: ['**/*.cjs'],
+      extends: ['eslint-config-upleveled'],
+    },
```

Comment:
command `pnpm eslint . --max-warnings 0`  cant find any match for the "." selector what causes the eslint command to fail, adding this configuration make sure eslint matches files in the root of the project. After some research this behavior comes from the react plugin we are using in the eslint config.

## without this line:

```console
➜  stylelint-config-upleveled git:(add-configuration) pnpm eslint . --max-warnings 0

Oops! Something went wrong! :(

ESLint: 8.37.0

No files matching the pattern "." were found.
Please check for typing mistakes in the pattern.
```

## adding the configuration

```console
➜  stylelint-config-upleveled git:(add-configuration) ✗ pnpm eslint . --max-warnings 0
Pages directory cannot be found at /home/josehower/upleveled-projects/stylelint-config-upleveled/pages or /home/josehower/upleveled-projects/stylelint-config-upleveled/src/pages. If using a custom path, please configure with the `no-html-link-for-pages` rule in your eslint config file.
Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.
```

the react warning can be ignored since it is not really causing the test to fail.
![image](https://user-images.githubusercontent.com/61600906/231980516-8dbbf75b-8c7d-4467-a654-b6235a219623.png)

Reply:
alright, and what is this `overrides` config actually doing? what are the alternatives you considered?

Reply:
https://github.com/upleveled/stylelint-config-upleveled/pull/3/commits/56425cb7aed9c377bea26b4b3843cabb5c10af88

Reply:
https://github.com/upleveled/stylelint-config-upleveled/pull/3/commits/f897b4e8c70210e53957ffa111fa43028926e89d